docs/meshcloud.payment-methods.md

```diff
@@ -11,6 +11,8 @@ Payment methods are created **per** meshCustomer and can be enhanced with [metad
 
 ![Payment Methods Lifecycle](assets/payment_methods/payment_method_overall_lifecycle.png)
 
+> Current limitation to how meshStack handles currencies: Although budgets can be applied in Dollar and Euro, budget alerts and payment method status only considers Euro. It is planned to expand the support for currencies besides Euro further in the future.
```

Comment:
d: I think we should also mention this limitation on the budget alert page: https://meshcloud-docs-dev-pr-598.onrender.com/docs/meshcloud.budget-alerts.html

Reply:
done